src/metabase/api/database.clj

```diff
@@ -983,10 +984,20 @@
   ;; just wrap this in a future so it happens async
   (let [db (api/write-check (t2/select-one Database :id id))]
     (events/publish-event! :event/database-manual-sync {:object db :user-id api/*current-user-id*})
-    (future
-      (sync-metadata/sync-db-metadata! db)
-      (analyze/analyze-db! db)))
-  {:status :ok})
+    (if-let [ex (try (do
```

Comment:
this do is redundant

Reply:
Fixed c8d2c626d9f3bbd551d059ebbdd87a3557ae48fe